catalogue_pipeline/ingestor/src/test/scala/uk/ac/wellcome/platform/ingestor/IngestorFeatureTest.scala

```diff
@@ -61,7 +61,7 @@ class IngestorFeatureTest
           .get
       )
 
-      val flags = sqsLocalFlags(queue) ++ esLocalFlags(indexName, itemType)
+      val flags = sqsLocalFlags(queue) ++ esLocalFlags(indexName, "", itemType)
```

Comment:
If we're going to start passing empty strings into places, using keyword arguments would be useful for clarity.

Reply:
That's empty because it's not using the second index yet, so it's irrelevant for now. Once I get to use it in tests it will have a variable with a meaningful name. For now I did the simplest thing to unbreak the tests that weren't relevant to the changes I was making